uberfire-structure/uberfire-structure-backend/src/test/java/org/guvnor/structure/backend/pom/PomEditorDefaultTest.java

```diff
@@ -37,15 +42,22 @@
 
     private final String POM = "pom.xml";
     private PomEditor editor;
+    private DynamicDependencyTypeConfigurationMap configurationMap;
     private Path tmpRoot, tmp;
+    private String JPA_HIBERNATE_VERSION;
 
     @Before
     public void setUp() throws Exception {
-        editor = new PomEditorDefault();
+        configurationMap = new DynamicDependencyTypeConfigurationMap();
+        editor = new PomEditorDefault(configurationMap);
         tmpRoot = Files.createTempDirectory("repo");
         tmp = TestUtil.createAndCopyToDirectory(tmpRoot,
                                                 "dummy",
                                                 "target/test-classes/dummy");
+        DependencyType depType = new FakeJPATypeDependency();
+        configurationMap.addDependencies(depType,
+                                         depType.getDependencies());
+        JPA_HIBERNATE_VERSION = configurationMap.getMapping().get(depType.getType()).get(0).getVersion();
```

Comment:
why hibernate test here?

Reply:
@porcelli Is a Fake dependency, I'll change the name